ui/components/LoadingComponents/LoadingComponentServer.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 // import AnimatedMeshery from './Animations/AnimatedMesheryCSS';
 import PropTypes from 'prop-types';
+// import AnimatedMeshery from './Animations/AnimatedMesheryCSS';
```

Comment:
Seems not to be related to current PR

Reply:
@lekaf974 I removed the unwanted file  guess it was due of rebase ,the comment get repeated